nautobot_circuit_maintenance/__init__.py

```diff
@@ -84,7 +84,12 @@ class CircuitMaintenanceConfig(PluginConfig):
     min_version = "1.0.0-beta.4"
     max_version = "1.999"
     required_settings = []
-    default_settings = {}
+    default_settings = {
+        "raw_notifications": {
+            "initial_days_since": 365,
+            "raw_notification_size": 8192,
+        },
+    }
```

Comment:
Does this make it so that if I change `raw_notification_size` in my `nautobot_config.py`, I have to also define a value for `initial_days_since` to avoid an error? It seems like setting:

```python
PLUGINS_CONFIG={
    "nautobot_circuit_maintenance": {
        "raw_notifications": {"raw_notification_size": 16384}
    }
}
```

would result in `settings["PLUGINS_CONFIG"]["nautobot_circuit_maintenance"]["raw_notifications"]["initial_days_since"]` becoming undefined.

Reply:
true